addons/l10n_do/data/account_account_tag.xml

```diff
@@ -1,99 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-    <data noupdate="1">
-
-
-    <!-- Account Tags -->
-
-        <record id="tag_R_ISR_10" model="account.account.tag">
```

Comment:
As I said: 
> You can rename the tags, but dont modify the xml-ids, and don't add or remove any.
The problem is: if you do that, you may break people current installation and force them to make an update of the module.

> Well, that's a bummer as we have a few missing there, and others that are not needed at all. I'll like to keep the old approach, but I'm actually on the side that both (community and enterprise) should be changed to meet the most common and practical cases.
You can do so, but then the 2 PR need to target master. And both the PR have to be done/merged together

So there are 2 solutions only: 
1) don't touch the xml-ids of the tags, just rename the field name if you want. This PR can be accepted in stable
2) do whatever you wanna do with the tags: you need to modify both the community and enterprise repo. We will only accept that in master if both PR are ready together